examples/mplot3d/trisurf3d_demo.py

```diff
@@ -27,6 +27,6 @@
 fig = plt.figure()
 ax = fig.gca(projection='3d')
 
-ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2)
+ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2, antialiased=True)
```

Comment:
we should probably get rid of the use of Jet here. Also, isn't antialiasing turned on by default?